### ISSUE TITLE
How Can I use my own voice to training the AI

### ISSUE THREAD

**Issue by CodeMagic6** [2noise/ChatTTS#284]
I want to use my own voice to training the AI, but I don't know the steps.Can anyone write a torturial about it?

我不知道怎么使用自己的语音训练AI生成自己的语音，有人能写出详细的步骤吗？

**Comment by cesinsingapore** [comment]
same question, i need the english speaker to speak more singlish/chinese accent

**Comment by kuang-kuang** [comment]
I also need clone my own voice

**Comment by MethanJess** [comment]
This fork is trying to add fine tuning: https://github.com/ain-soph/ChatTTS

**Comment by yoesak** [comment]
> This fork is trying to add fine tuning: https://github.com/ain-soph/ChatTTS

Does it support to train a new language? instead of fine-tune for cloning voice?

**Comment by cesinsingapore** [comment]
> > This fork is trying to add fine tuning: https://github.com/ain-soph/ChatTTS
> 
> Does it support to train a new language? instead of fine-tune for cloning voice?

the fork is not clear how to do the fine tune

**Comment by xpdd123** [comment]
> > > This fork is trying to add fine tuning: https://github.com/ain-soph/ChatTTS
> > 
> > 
> > Does it support to train a new language? instead of fine-tune for cloning voice?
> 
> the fork is not clear how to do the fine tune

first finetune encoder, then finetune gpt

**Comment by vivek-kumar-vkb** [comment]
> I’m the one who create that fork. Yes, first train the encoder and then GPT. However, the current encoder loss cannot be optimized to be smaller than 0.1, making training unfeasible.
> 
> The possible solution is to modify current VQ encoder architecture and explore better training hyper-parameters. Or find a better dataset for training.
> 
> But I’m busy working, so … Hope anyone other could do that

@ain-soph Can you tell me how to fine-tune encoder and then GPT? Unfortunately I am not able to find script of finetuning anywhere. If you could guide me in this it would be great.
